reactquiz/src/components/quiz/Results.js

```diff
@@ -1,34 +1,25 @@
-import React,{Component} from 'react';
-
-
-class Results extends Component{
-
+import React, { Component } from 'react';
 
+class Results extends Component {
 	render() {
-
-		var percent =(this.props.score / this.props.questions.length*100);
-		if (percent > 80){
-
-			var message ='Awesome Job!';
-		} else if(percent < 80 && percent >60)
-		{
-			var message = 'You did Ok !';
-		}
-		else {
-
-			var message ='Not satisfactory. Should try once again !';
+		const percent = (this.props.score / this.props.questions.length * 100 * 100);
+		let message;
+		if (percent > 80) {
+			message = 'Awesome Job!!';
+		} else if (percent > 60) {
+			message = 'You did Ok!';
+		} else {
+			message = 'Not satisfactory. Should try once again!!!';
 		}
-		return(
+		return (
 			<div className="well">
-				<h4> You got {this.props.score} out of {this.props.questions.length} Correct</h4>
+				<h4>You got {this.props.score} out of {this.props.questions.length} Correct</h4>
 				<h1>{percent}% - {message}</h1>
 				<hr />
-				<a href ="https://harshil562.github.io/reactquiz/app/"> Take again </a>
-
+				<a href="https://harshil562.github.io/reactquiz/app/">Take again</a>
 			</div>
-		)
+		);
 	}
 }
 
```

Comment:
The code logic looks good. However, I have a few suggestions about the code style:

1.  For better readability, consider using template literals for the `<h1>` tag content. For example:
    ```javascript
    <h1>`${percent}% - ${message}`</h1>
    ```
2.  There are some inconsistencies with spaces in your code. For example, sometimes you have spaces around operators and sometimes you don't. It would be good to stick to a consistent style.
3.  The comment `//some apis will going to expose it's json code to us , then only we can get the database access` seems irrelevant to this file. You can remove it.